docs/contributing/coding-guidelines/ros-nodes/directory-structure.md

```diff
@@ -4,6 +4,8 @@ This document describes the directory structure of ROS nodes within Autoware.
 
 We'll use the package `autoware_gnss_poser` as an example.
 
+**Note that this example does not reflect the actual `autoware_gnss_poser`, and has extra files and directories to demonstrate all posssible package structure.**
```

Comment:
There seems to be a typo. ("posssible")

Reply:
Thanks for pointing it out!
Fixed in here. https://github.com/autowarefoundation/autoware-documentation/pull/601/commits/042c17a3ab1cbae8b056e278b6173dda7f95a045